src/repl/scala/tools/nsc/interpreter/package.scala

```diff
@@ -199,10 +199,11 @@ package object interpreter extends ReplConfig with ReplStrings {
   }
 
   /* debug assist
+  debug assist */
   private[nsc] implicit class `smart stringifier`(val sc: StringContext) extends AnyVal {
     import StringContext._, runtime.ScalaRunTime.stringOf
     def ss(args: Any*): String = sc.standardInterpolator(treatEscapes, args map stringOf)
-  } debug assist */
+  }
```

Comment:
Thx, after a couple of clean-up push -f, I was too tired when I saw that one. I figured to wait until a follow-up...